pyteal/ast/subroutine.py

```diff
@@ -589,6 +589,10 @@ def __call__(
     def name(self) -> str:
         return self.subroutine.name()
 
+    def method_signature(self) -> str:
+        args = [str(v) for v in self.subroutine.abi_args.values()]
+        return f"{self.name()}({','.join(args)}){self.type_of()}"
+
```

Comment:
Not sure how this made it in 🤔, should I just leave it? @ahangsu guessing this will take some manual wiggling when feature/abi gets merged back into router

Reply:
confirmed it is from abi-router branch, is it needed in this branch? If so we can open a minor PR to include the change to feature/abi?

Reply:
not necessary, just removed it